Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -28,11 +28,17 @@ class PublishTimePeriodExtension extends AdminExtension
     protected $formGroup;
 
     /**
+     * @var string
+     */
+    protected $formTab;
+
+    /**
      * @param string $formGroup - group to use for form mapper
      */
-    public function __construct($formGroup = 'form.group_publish_workflow')
+    public function __construct($formTab = 'form.tab_publish', $formGroup = 'group_publish_workflow')
```

Comment:
if you would add the tab after the group, we would avoid a BC break in the constructor. in case somebody extended this class...